api-participant/src/main/java/bio/terra/pearl/api/participant/controller/PortalController.java

```diff
@@ -46,4 +50,41 @@ public ResponseEntity<Object> listPortalEnvAlerts(String portalShortcode, String
     return ResponseEntity.ok(
         portalDashboardConfigService.findByPortalEnvId(portalEnvironment.getId()));
   }
+
+  @Override
+  @CrossOrigin(
+      origins = {
+        "https://juniperdemodev.b2clogin.com", // Heart (demo only)
+        "https://junipercmidemo.b2clogin.com", // CMI (demo only)
+        "https://ourhealthdev.b2clogin.com", // OurHealth (demo)
+        "https://ourhealthstudy.b2clogin.com", // OurHealth (prod)
+        "https://hearthivedev.b2clogin.com", // HeartHive (demo)
+        "https://hearthive.b2clogin.com", // HeartHive (prod)
+        "https://gvascdev.b2clogin.com", // gVASC (demo)
+        "https://gvascprod.b2clogin.com" // gVASC (prod)
+      },
+      maxAge = 3600,
+      methods = {RequestMethod.GET, RequestMethod.OPTIONS})
```

Comment:
The `@CrossOrigin` annotation doesn't support reading the origins from config, so for now we're just going to roll with the hardcoded list. It will get unwieldy at some point, and then we'll work out a more creative solution. We're not going to do demo + prod tenants for every new customer, so the list shouldn't grow _super_ quickly.

Reply:
let's add a comment to this method that it is only returning publicly-available assets